packages/ui-form-field/src/FormField/index.tsx

```diff
@@ -66,7 +66,7 @@ class FormField extends Component<FormFieldProps> {
       <FormFieldLayout
         {...omitProps(this.props, FormField.allowedProps)}
         {...pickProps(this.props, FormFieldLayout.allowedProps)}
-        // @ts-expect-error ts-migrate(2322) FIXME: Remove this comment to see the full error message
+        label={this.props.label}
```

Comment:
It was passed from the pickProps, but had to add it manually because TS was complaining (it cannot see what comes back from pickProps)